api/app/controllers/spree/api/v2/storefront/cart_controller.rb

```diff
@@ -35,10 +37,10 @@ def add_item
             result = add_item_service.call(
               order: spree_current_order,
               variant: @variant,
-              quantity: add_item_params[:quantity],
-              public_metadata: add_item_params[:public_metadata],
-              private_metadata: add_item_params[:private_metadata],
-              options: add_item_params[:options]
+              quantity: params[:quantity],
```

Comment:
@aleksandarpetrushev we don't want to use raw params, always use permitted params

Reply:
@damianlegawiec Ok, btw it was with raw params before, then I used permitted add_item_params for whitelisting the metadata params to be able to cast them into hash for the set metadata when adding item to cart. Then I needed the same (to cast them) for this task and just whitelisted the metadata params instead of all of them. But I will fix it